google/ads/google_ads/interceptors/logging_interceptor.py

```diff
@@ -117,7 +117,10 @@ def _get_customer_id(self, request):
         Args:
             request: An instance of a request proto message.
         """
-        return getattr(request, 'customer_id', None)
+        if hasattr(request, 'customer_id'):
+            return getattr(request, 'customer_id')
```

Comment:
The Ruby version returns "N/A" (which is probably wrong since '/' is the delimiter for resource name parts). In Python, perhaps you should a default value if none is found, ie. "NONE" or "UNKNOWN"

Reply:
I think we thought about how to do this a long time ago when logging was first implemented and went with `None` since it will serialize into the string `'None'` when printed. In any case I don't think it was arbitrary to return `None` here initially and I'd prefer to stick with that.

Reply:
SGTM